src/js/tabs/coldwalletsettings.js

```diff
@@ -24,23 +24,6 @@ ColdwalletSettingsTab.prototype.angular = function(module) {
     if (!id.loginStatus) {
       id.goId();
     }
-    if ($scope.userBlob.data.defaultDirectory) {
```

Comment:
I think the user should still be able to change the default directory from this page if they want to later on, but the function is moved to app.js